packit/config/package_config.py

```diff
@@ -417,6 +427,19 @@ def get_local_package_config(
 
     loaded_config = load_packit_yaml(config_file_path=config_file_name)
 
+    try:
+        local_config_file_name = find_packit_yaml(
+            *directory,
+            try_local_dir_first=True,
+            try_local_dir_last=False,
+            suffix="Local",
```

Comment:
I have a feeling that this is a pretty good candidate for extending the part of the user config, so it's possible to configure this. You have prepared it pretty well for that.